xknx/devices/switch.py

```diff
@@ -50,7 +50,8 @@ def has_group_address(self, group_address):
     @property
     def state(self):
         """Return the current switch state of the device."""
-        return self.switch.value == RemoteValueSwitch.Value.ON
+        # None will return False
+        return bool(self.switch.value)
```

Comment:
Usually switch and light.state (and probably climate but there is no such test for the on state) is initialized with `None`. Until now `return self.switch.value == RemoteValueSwitch.Value.ON` has returned `False` for `None`.

Technically I would say the new implementation is correct - we don't know the state so it is None but I have no idea if this breaks things. 

Alternatively we could return `False` for `None` (in the device)
```
def state(self):
    return bool(self.device.value)
```